pkg/api/customization/cluster/formatter.go

```diff
@@ -47,23 +47,21 @@ func (f *Formatter) Formatter(request *types.APIContext, resource *types.RawReso
 			resource.AddAction(request, v3.ClusterActionBackupEtcd)
 			resource.AddAction(request, v3.ClusterActionRestoreFromEtcdBackup)
 		}
-		canUpdateCluster := func(request *types.APIContext, clusterID string) bool {
+		canUpdateClusterFn := func(request *types.APIContext, clusterID string) bool {
 			cluster := map[string]interface{}{
 				"id": clusterID,
 			}
-			return request.AccessControl.CanDo(v3.ClusterGroupVersionKind.Group, v3.ClusterResource.Name, "update", request, cluster, resource.Schema) == nil
-		}
-		canCreateClusterScanObject := func() bool {
 			return request.AccessControl.CanDo(
-				v3.ClusterScanGroupVersionKind.Group,
-				v3.ClusterScanResource.Name,
-				"create",
+				v3.ClusterGroupVersionKind.Group,
+				v3.ClusterResource.Name,
+				"update",
 				request,
-				nil,
-				request.Schema,
-			) == nil
+				cluster,
+				resource.Schema) == nil
 		}
-		if canUpdateCluster(request, resource.ID) && canCreateClusterScanObject() {
+		canUpdateCluster := canUpdateClusterFn(request, resource.ID)
+		logrus.Debugf("user: %v, canUpdateCluster: %v", request.Request.Header.Get("Impersonate-User"), canUpdateCluster)
+		if canUpdateCluster {
```

Comment:
I'm just missing something with this fix that makes me not understand it. What bug are you fixing by making this check less restrictive? Is the TL;DR that you don't need to be able to create scan objects in order to run a scan? That doesn't seem right?

Reply:
In an earlier PR, `Manage Cluster Scan` role was removed, hence there is no need for a redundant check for that role. With the current requirement, only a cluster-owner needs to be able to run the scan and since a cluster can be updated by cluster-owner using that check.

Reply:
ok makes sense now ty